openshift-ci/Dockerfile.tools

```diff
@@ -1,23 +1,29 @@
 FROM fedora:31
+
 ENV GOPATH /go
 ENV GOBIN /go/bin
 ENV GOCACHE /go/.cache
 ENV PATH=$PATH:$GOPATH/bin
-ENV GOVERSION golang-1.13.5-1.fc31
+ENV GOVERSION 1.13.5
 ARG GO_PACKAGE_PATH=github.com/openshift-kni/cnf-features-deploy
 
 # rpms required for building and running test suites
 RUN dnf -y install \
-    $GOVERSION \
+    git \
     make \
-    tar \
```

Comment:
I think we still need tar

Reply:
it installed as part of some other package